src/transformers/__init__.py

```diff
@@ -3357,6 +3370,7 @@
     from .models.encoder_decoder import EncoderDecoderConfig
     from .models.ernie import ERNIE_PRETRAINED_CONFIG_ARCHIVE_MAP, ErnieConfig
     from .models.esm import ESM_PRETRAINED_CONFIG_ARCHIVE_MAP, EsmConfig, EsmTokenizer
+    from .models.fan import FAN_PRETRAINED_CONFIG_ARCHIVE_MAP, FANConfig, FANTokenizer
```

Comment:
```suggestion
    from .models.fan import FAN_PRETRAINED_CONFIG_ARCHIVE_MAP, FANConfig
```
FAN doesn't have a tokenizer, I assume